plopfile.js

```diff
@@ -338,6 +353,45 @@ module.exports = (plop) => {
         }
       }
 
+      if (!exclusive || platforms === "windows") {
+        if (isInstalled("react-native-windows", exclusive)) {
+          const prefix = exclusive ? "" : "windows/";
+          actions.push({
+            type: "add",
+            path: `${prefix}test-app.bat`,
+            template: `node ./node_modules/react-native-test-app/windows/test-app.js --projectDirectory=${
+              exclusive ? "./" : "windows/"
+            }`,
+          });
```

Comment:
Do we still need this?

Reply:
Should maybe add this to `postinstall` in `package.json`?

Reply:
On the other hand, `resources` will always be empty, so maybe not. Regardless, I don't think we need the `.bat` file is the script is installed.